W-15133046 oms1.1 (#137)
mulesoft/docs-accelerators: salesforce-oms/1.1/modules/ROOT/pages/sfdc-oms-setup-customization-guide.adoc

* Disable the *Enable Reduction Orders* option in *Order Settings*.
* Enable *Order Management*, *Use Person Accounts*, and *High Scale Orders*
* Activate *Multiple Currencies*
* Enable *State* and *Country/Territory* picklists
* <<configure_payment_gateway>>
* Configure either a digital wallet or credit card payment method and associate an account with it

== Customize Salesforce OMS

Install the Salesforce package from the https://anypoint.mulesoft.com/exchange/997d5e99-287f-4f68-bc95-ed435d7c5797/accel-b2b-common-resources-src/[Accelerator B2B Common Resources] project and verify that the configuration is correct in your Salesforce organization.

Follow these steps in Salesforce OMS after installing the package successfully:

> * Create a matching rule named `Account Interchange Code Match` in *Setup > Matching Rules*. Enter `Account: B2B_EDISenderCode EXACT MatchBlank = FALSE` in the *Matching Criteria* field.

Reviewer: ```suggestion
. Create a matching rule named `Account Interchange Code Match` in *Setup > Matching Rules*. Enter `Account: B2B_EDISenderCode EXACT MatchBlank = FALSE` in the *Matching Criteria* field.
```

> * Create a duplicate rule named `Account Interchange Code Rule` in *Setup > Duplicate Rules*. Select `Account Interchange Code Match` as the matching rule. After you save and activate the duplicate rule, open the rule in  *Duplicate Management > Duplicate Rules* and set the order to 1 in the *Reorder* field, which gives the rule precedence over the standard *Account* object matching rule.

Reviewer: ```suggestion
. Create a duplicate rule named `Account Interchange Code Rule` in *Setup > Duplicate Rules*. Select `Account Interchange Code Match` as the matching rule. After you save and activate the duplicate rule, open the rule in  *Duplicate Management > Duplicate Rules* and set the order to 1 in the *Reorder* field, which gives the rule precedence over the standard *Account* object matching rule.
```

* Disable the *Enable Reduction Orders* option in *Order Settings*.
* Enable *Order Management*, *Use Person Accounts*, and *High Scale Orders*
* Activate *Multiple Currencies*
* Enable *State* and *Country/Territory* picklists
* <<configure_payment_gateway>>

> * Setup a Digital Wallet or Credit Card payment method 

Reviewer: ```suggestion
* Configure either a digital wallet or credit card payment method and associate an account with it
```

> * Associate Account with payment method

Reviewer: ```suggestion
```

> * Create a new Digital Wallet from the Digital Wallets page if there is no Payment Method already configured for the Account. A Digital Wallet can be created by filling in the Account, Processing Mode, Nickname and Type fields. 

Reviewer: This implies that you're required to create a digital wallet payment method, but the bullet above indicates you can create a credit card payment method instead. Is a digital wallet required?

Author: What it means, create either digital wallet or Credit Card payment method but if there is no payment method available create digital wallet.

Reviewer: What makes the option to create a credit card payment method not available?

Author: if its not setup.

Reviewer: To clarify, you're saying that the option to create a digital wallet payment method is always available? Is there a setting in the application that enables the credit card payment method option to be available to set up as well?

Author: @tasmoore  there is no help document available for creating Digital wallet or card payment.

> * Create a new Digital Wallet from the Digital Wallets page if there is no Payment Method already configured for the Account. A Digital Wallet can be created by filling in the Account, Processing Mode, Nickname and Type fields. 

Reviewer: ```suggestion
```

> == Customization

Reviewer: ```suggestion
== Customize Salesforce OMS
```

Install the Salesforce package from the https://anypoint.mulesoft.com/exchange/997d5e99-287f-4f68-bc95-ed435d7c5797/accel-b2b-common-resources-src/[Accelerator B2B Common Resources] project and verify that the configuration is correct in your Salesforce organization.

> Once the package is installed successfully, proceed with the following steps:

Reviewer: ```suggestion
Follow these steps after installing the package successfully:

```

> Follow these steps after installing the package successfully:

Reviewer: ```suggestion
Follow these steps in Salesforce OMS after installing the package successfully:
```

> * Create matching Rule Account Interchange Code Match with Matching Cretaria Account: B2B_EDISenderCode EXACT MatchBlank = FALSE 

Reviewer: ```suggestion
* Create a matching rule named `Account Interchange Code Match` in *Setup > Matching Rules*. Enter `Account: B2B_EDISenderCode EXACT MatchBlank = FALSE` in the *Matching Criteria* field.
```

> * Create duplicate rule named 'Account Interchange Code Rule' with the matching rule 'Account Interchange Code Match'. Set the order to 1, giving it precedence over the standard account matching rule.

Reviewer: I don't understand these steps. Where do you perform them?

Author: We need to create a matching rule named 'Account Interchange Code Match' with matching criteria "B2B_EDISenderCode EXACT MatchBlank = FALSE"

Author: these needs to done on the OMS Rule Setup

Reviewer: Is "OMS Rule" a page name in Setup?

Author: ![Screenshot 2024-06-12 at 2 07 41 PM](https://github.com/mulesoft/docs-accelerators/assets/48571056/6a4ff1db-aeac-4b9a-bf0d-59684a05612f) this is a common setup inside Salesforce and developers/admin knows it

> * Create duplicate rule named 'Account Interchange Code Rule' with the matching rule 'Account Interchange Code Match'. Set the order to 1, giving it precedence over the standard account matching rule.

Reviewer: ```suggestion
* Create a duplicate rule named `Account Interchange Code Rule` in *Setup > Duplicate Rules*. Select `Account Interchange Code Match` as the matching rule. After you save and activate the duplicate rule, open the rule in  *Duplicate Management > Duplicate Rules* and set the order to 1 in the *Reorder* field, which gives the rule precedence over the standard *Account* object matching rule.
```

Author: @tasmoore We need to add the matching criteria Account: B2B_EDISenderCode EXACT MatchBlank = FALSE

|Shipment |ExternalId |ExternalId__c |Text |100 |N |Y
|===

[NOTE]

> You must add the *Read* and *Write* permissions to the custom fields created by Salesforce package.

Reviewer: ```suggestion
You must add the *Read* and *Write* permissions to the custom fields created by the Salesforce package.
```

|Shipment |ExternalId |ExternalId__c |Text |100 |N |Y
|===

> IMPORTANT: Add read/ write permissions to custom fields created by Salesforce package.

Reviewer: Is there a permission name?

Author: yes

> === Custom Object Tab

Reviewer: ```suggestion
=== Verify You Have the Custom Object Tab
```

|Shipment |ExternalId |ExternalId__c |Text |100 |N |Y
|===

> IMPORTANT: Add read/ write permissions to custom fields created by Salesforce package.

Reviewer: ```suggestion
[NOTE]
You must add the *Read* and *Write* permissions to the custom fields created by Salesforce package.
```
